harness/features/allVariables.cloud.test.ts

```diff
@@ -61,15 +62,24 @@ describe('allVariables Tests - Cloud', () => {
         it('should return a variable map', async () => {
             scope
                 .post(`/client/${client.clientId}/v1/variables`)
-                .reply(200, variables)
+                .reply(
+                    200,
+                    getMockedVariables(
+                        hasCapability(sdkName, Capabilities.variablesFeatureId),
```

Comment:
Why not call this what it actually is - reduced event payload size?

Reply:
that's not what this is testing, its testing that `_feature` param is added to the `variables` array for the bucketed config.

Reply:
Eh I guess. Just a weird name